packages/web3-eth-contract/test/integration/contract_methods_errors.test.ts

```diff
@@ -75,6 +75,27 @@ describe('contract errors', () => {
 			});
 		});
 
+		it('should catch Unauthorized error PromiEvent.on("error")', async () => {
+			const expectedThrownError = {
```

Comment:
I think it would be nice if we have this `const expectedThrownError = {...` above and we use it inside the 2 related test cases which related to the `Unauthorized()` error.